src/app/app-extras.module.ts

```diff
@@ -3,18 +3,17 @@ import {
 } from '@angular/core';
 
 import {
-  MyLibraryModule
+  SkySplitViewModule
 } from './public';
 
-// Specify entry components, module-level providers, etc. here.
+import {
+  AppSkyModule
```

Comment:
I'm confused. This was approved in a prior PR: https://github.com/blackbaud/skyux-split-view/pull/2
Has something changed since then?

Also, split-view shouldn't be providing things like repeater - that's only used for the demo. We could revert the state of this file back to where it was prior to that PR ☝️ ?

Reply:
Whoops, that definitely should have been caught (by me). Sorry about that. I've been removing that module in all other library migrations. For example: https://github.com/blackbaud/skyux-core/pull/84/files

Reply:
Yeah, for modules that are needed by the visual demo, we can just add those to the `exports` array.

Reply:
This has been updated.